leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java

```diff
@@ -995,12 +1049,12 @@ private void setNewLocation(Address address) {
       currentLocation = new Location(getString(R.string.network_resource));
     }
 
-    double latitude = address.getLatitude();
-    double longitude = address.getLongitude();
+    double lat = address.getLatitude();
```

Comment:
i was working with lat and long, and later I decided to rename all variables to be more uniform through out the lib